CMakeLists.txt

```diff
@@ -757,11 +757,11 @@ endforeach(sourcefile ${C_TEST_EXES})
 add_subdirectory(tools)
 
 # Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
Yes. This is useful to be used together with ExternalProject.cmake module or Hunter package manager.